tools/sicer/sicer_wrapper.xml

```diff
@@ -1,155 +1,156 @@
-      <param name="fix_off_by_one_errors" value="True" />
-      <param name="redundancy_threshold" value="1" />
-      <param name="window_size" value="200" />
-      <param name="fragment_size" value="150" />
-      <param name="effective_genome_fraction" value="0.74" />
-      <param name="gap_size" value="600" />
-      <param name="error_cut_off" value="0.01" />
-      <output name="redundancy_removed_test_bed_output_file" file="test-1-removed.bed" />
-      <output name="redundancy_removed_control_bed_output_file" file="control-1-removed.bed" />
-      <output name="summary_graph_output_file" file="test_3_test-W200.graph" />
-      <output name="test_normalized_wig_output_file" file="test-W200-normalized.wig" />
-      <output name="significant_islands_output_file" file="test_3_test-W200-G600-FDR0.01-island.bed" />
-      <output name="island_filtered_output_file" file="test-W200-G600-FDR0.01-islandfiltered.bed" />
-      <output name="island_filtered_normalized_wig_output_file" file="test-W200-G600-FDR0.01-islandfiltered-normalized.wig" />
-      <output name="score_island_output_file" file="test_3_test-W200-G600.scoreisland" />
-      <output name="islands_summary_output_file" file="test_3_test-W200-G600-islands-summary" />
-      <output name="significant_islands_summary_output_file" file="test_3_test-W200-G600-islands-summary-FDR0.01" />
-      <output name="output_log_file" file="test_2_output_log_file.contains" compare="contains"/>
-    </test>
-    <test>
-      <param name="input_bed_file" value="chipseq_enriched.bed.gz" ftype="bed" dbkey="mm8" />
-      <param name="input_control_file" />
-      <param name="fix_off_by_one_errors" value="True" />
-      <param name="redundancy_threshold" value="1" />
-      <param name="window_size" value="200" />
-      <param name="fragment_size" value="150" />
-      <param name="effective_genome_fraction" value="0.74" />
-      <param name="gap_size" value="600" />
-      <param name="error_cut_off" value="0.01" />
-      <output name="redundancy_removed_test_bed_output_file" file="test-1-removed.bed" />
-      <output name="summary_graph_output_file" file="test_3_test-W200.graph" />
-      <output name="test_normalized_wig_output_file" file="test-W200-normalized.wig" />
-      <output name="island_filtered_output_file" file="test-W200-G600-E0.01-islandfiltered.bed" />
-      <output name="island_filtered_normalized_wig_output_file" file="test-W200-G600-E0.01-islandfiltered-normalized.wig" />
-      <output name="score_island_output_file" file="test_4_test-W200-G600-E0.01.scoreisland" />
-      <output name="output_log_file" file="test_1_output_log_file.contains" compare="contains"/>
-    </test>
-  </tests>
-  <help>
+    <description>Statistical approach for the Identification of ChIP-Enriched Regions</description>
+    <requirements>
+        <requirement type="package" version="1.1">SICER</requirement>
```

Comment:
```suggestion
        <requirement type="package" version="1.1">sicer</requirement>
```
Nice find, but I think this should be lowercase since the bioconda package and docker images are lowercase (uppercase docker images aren't valid either, xref: https://github.com/galaxyproject/galaxy/pull/9733).

Reply:
In general I agree, but on the other hand this has been a nice test case and I don't want to increase the tool version just for that.

Reply:
I would say the tool version should be increased, the tempfile handling has changed and it's now Python 3 compatible.

Reply:
It's not changing requirements, tool inputs or outputs, so I'd call all these "internal changes", but I could increase the tool version nonetheless if you let me keep the uppercase requirement as a test case :D

Reply:
I'm going to merge this, hope that's OK.